### ISSUE TITLE
Reduced collaborative play

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7323]
**Preferred term label:**
Reduced collaborative play

**Synonyms**

**Definition (free text, please give PubMed ID)**
Impairment in a child ability to be able to play in a way that involves peer collaboration. Collaborative play involves things such as verbal and non-verbal communication  (e.g.,  self-initiated interactions),  turn-taking,  sharing,  reciprocity,  and collaborative problem-solving. It is often contrasted to parallel play, which involves a form of social play in which two children do similar activities near each other and often mimic each other. Typically developing children develop collaborative play between the ages of 4 and 5.
PMID: 25428293

**Parent term (use hpo.jax.org/app)**
Impaired social interactions
HP:0000735

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
added as HP:4000082
